src/main/webapp/app/exercises/text/participate/text-editor.component.ts

```
@@ -67,13 +71,23 @@
     // indicates if the assessment due date is in the past. the assessment will not be loaded and displayed to the student if it is not.
     isAfterAssessmentDueDate: boolean;
     examMode = false;
-
+    isGeneratingFeedback = false;
     // indicates, that it is an exam exercise and the publishResults date is in the past
     isAfterPublishDate: boolean;
     isOwnerOfParticipation: boolean;
-
+    isReadOnlyWithShowResult: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)